src/include/miopen/conv/kernel_interface/winograd_kernel_interface.hpp

```diff
@@ -26,6 +26,10 @@
 
 #pragma once
 
+#include <type_traits>
+#include <ostream>
+#include <stdint.h>
```

Comment:
```suggestion
#include <cstdint>
```

Reply:
On Windows, `<cstdint>` puts those types in the `std` namespace. So, instead of `uint32_t`, we should use `std::uint32_t`. I wanted to minimize the changes required.